collectors/apicmcpfault.py

```diff
@@ -0,0 +1,58 @@
+import logging, re
+
+from prometheus_client.core import CounterMetricFamily, Summary
+import BaseCollector
+
+LOG = logging.getLogger('apic_exporter.exporter')
+REQUEST_TIME = Summary('apic_ips_processing_seconds',
```

Comment:
This should not be `apic_ips_processing_seconds` but more something like `apic_mcp_processing_seconds`. The former is already used for the apic_ips metric.

Reply:
updated